stable/minio/values.yaml

```diff
@@ -72,6 +71,15 @@ defaultBucket:
 ## https://docs.minio.io/docs/minio-bucket-notification-guide
 ##
 minioConfig:
+  region: ""
```

Comment:
The current value for `region` is `us-east-1`. Won't changing it to an empty value break some installs when they update?

Reply:
Good catch @mattfarina I agree, this may cause failures, if client has assumed that server region is `us-east-1`. 

Updated the PR to fix this and the test failures.